pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -82,6 +82,21 @@ type GraphBuilder struct {
 	// dependencyGraphBuilder
 	monitors    monitors
 	monitorLock sync.RWMutex
+	// stoppedResources is the set of resources that have been uninstalled
+	// from the cluster in between gc Syncs.
+	// When the resource is uninstalled the shared informer for the resource
+	// (as well as the underlying controller and reflector) are stopped.
+	// We need to track these resources because if they are reinstalled before
+	// the next Sync, the garbage collector will be unaware that the informer
+	// has been stopped and will not restart the informer.
```

Comment:
this bit doesn't seem right... syncMonitors should always leave us with running monitors for the passed-in resources, right?

Reply:
fixed, see above explanation